ITKImageProcessingFilters/ImportRegisteredImageMontage.h

```diff
@@ -192,7 +192,7 @@ class ITKImageProcessing_EXPORT ImportRegisteredImageMontage : public AbstractFi
    */
   ITK_IMAGE_READER_HELPER_ImageDataArrayName() ITK_IMAGE_READER_HELPER_DECL()
 
-      public : ImportRegisteredImageMontage(const ImportRegisteredImageMontage&) = delete; // Copy Constructor Not Implemented
+      public : public : public : public : ImportRegisteredImageMontage(const ImportRegisteredImageMontage&) = delete; // Copy Constructor Not Implemented
```

Comment:
This is a bit too public for me.  Try a line break and delete 3 access modifiers.